components/ReadSpeaker/src/index.scss

```diff
@@ -1280,13 +1333,12 @@ dialog#rs-dialog {
           }
         }
         .rsmpl-playpause {
-          button[title="play"] {
-            @include readspeaker-icon-styles(var(--denhaag-readspeaker-icons-play));
-          }
+          button[title="play"],
```

Comment:
Zijn er andere attributen waarop je kan targeten? Als ze straks DE of FR willen supporten werkt dit niet meer?

Reply:
Nee helaas niet, zie screenshot. 
Maar ik zag net wel met het testen dat er toevallig nu een [title="play"] en ook een [title="Play"] in zit. 
Ik heb de PR geupdate en die toegevoegd.. Maar het zou goed kunnen dat we FR en EN straks ook nog moeten updaten, dat kan ik nu alleen niet testen

![Screenshot 2023-05-30 at 09 04 23](https://github.com/nl-design-system/denhaag/assets/131674048/400b1c6e-8049-4efa-a2b4-57760179f825)